schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+{
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Instrument Specification Definition",
+   "description": "The Instrument Specification Definition for Astro CSM",
+   "type": "object",
+
+   "properties": {
+       "BORESIGHT": {
+	   "description": "Boresight (x,y,z) vector",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "description": "Component of the boresight vector",
+		 "minItems": 3,
+                 "maxItems": 3
+            }
+       },
+       "DETECTOR_CENTER": {
+           "description": "Center of the detector in pixel coordinates",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 2,
+                 "maxItems": 2
+            }
+       },
+       "CENTER_EPHEMERIS_TIME": {
+           "description": "The center ephemeris time of the image",
+           "type": "number"
+       },
+       "STARTING_EPHEMERIS_TIME": {
+           "description": "The starting ephemeris time of the image",
+           "type": "number"
+       },
+       "FOCAL_LENGTH_MODEL": {
+            "description":  "A key to get information about time/temp dependent focal length models",
+            "type": "object",
+            "properties": {
+                "key": {"type": "string"},
+                "focal_length": {"type": "number"},
+                "focal_epsilon":{"type": "number"}
+            },
+   	"required": ["focal_length"]
+       },
+       "IFOV": {
+           "description": "Instantaneous field of view in degrees",
+           "type": "number"
+       },
+       "IMAGE_LINES": {
+           "description": "The number of lines in the image",
+           "type": "number"
+       },
+       "IMAGE_SAMPLES": {
+           "description": "The the number of samples in the image",
+           "type": "number"
+       },
+
+       "RADII": {
+           "description": "The radii of the target.",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 1,
+                 "maxItems": 3
```

Comment:
CSM currently only supports 2, so we should probably cap that here.

Reply:
Good point. I remember discussion about supporting triaxial ellipsoids, but did not remember the conclusion / where this discussion left off.

Reply:
Agreed capped at two for now in the v3.0.3 spec. Not that this will make everyone comforted but our current explanation: https://github.com/USGS-Astrogeology/CSM-CameraModel/wiki/CSM-and-triaxial